manifest.json

```
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store-graphql",
-  "version": "2.173.1",
+  "version": "2.173.2",
```

It is best to rely on the `vtex-io-ci-cd` bot to update the version number. 
Can you please revert the change to this file?

```suggestion
  "version": "2.173.1",
```